pkg/api/convertertoapi.go

```diff
@@ -376,13 +379,57 @@ func convertV20160330OrchestratorProfile(v20160330 *v20160330.OrchestratorProfil
 
 func convertV20170131OrchestratorProfile(v20170131 *v20170131.OrchestratorProfile, api *OrchestratorProfile) {
 	api.OrchestratorType = OrchestratorType(v20170131.OrchestratorType)
+	if api.OrchestratorType == Kubernetes {
+		api.OrchestratorVersion = Kubernetes162
```

Comment:
Right now, the testdata json and example json all take default OrchestratorVersion? Can we update some so the user has the example of input?